experimental/kubernetes/README.md

```diff
@@ -37,14 +27,118 @@ can authenticate with the cluster you want to deploy Spinnaker to.
 Once that is all squared away, make sure that running `$ kubectl config
 current-context` refers to the cluster you want to have Spinnaker running in.
 (You may need to update `kubectl` for the previous command to work).
+There are two paths to take here
+
+## Configuring your Docker registry
+
+Once you've decided which registries you will use and they are configured,
+please make sure that the following list in `./config/clouddriver-local.yml` is
```

Comment:
I might add a note that `./config/` is whichever config directory they're using, i.e. `.spinnaker` or `/opt/spinnaker/config`, etc.